crates/core/src/cf/writer.rs

```diff
@@ -416,8 +415,8 @@ mod tests {
 	}
 
 	async fn init(store_diff: bool) -> Datastore {
-		let dns = DefineNamespaceStatement {
-			name: crate::sql::Ident(NS.to_string()),
+		let dns = Namespace {
+			name: NS.to_string(),
 			..Default::default()
 		};
```

Comment:
Note to self: Still need to decide how `definition` should be populated in this case.